### ISSUE TITLE
Set Watched and record progress in kodi library when watched through Fen

### ISSUE THREAD

**Issue by Jm7698** [Tikipeter/repository.tikipeter#3]
add functionality to Set Watched in kodi library and record progress in kodi library and fen when watched in Fen or in Tmdbhelper fen player also regardless of watched  indicator (preferably trakt)

**Comment by Tikipeter** [comment]
TMDbHelper is not supported by Fen.

**Comment by Jm7698** [comment]
it doesn't record progress in kodi library even without tmdbhelper

**Comment by Tikipeter** [comment]
Let me rephrase.

Fen does not support library integration of any kind.

**Comment by Jm7698** [comment]
I understand, maybe you will reconsider adding it someday. it would be useful for many. 
But what about at least record progress in Fen if indicator is set to trakt, because many people using trakt also ?

**Comment by Tikipeter** [comment]
Trakt indicator progress is already recorded and shown in Fen.
Do you mean shown in Fen when played back from a library item? If so, that would be shown after the next time Fen syncs with Trakt.

If you understand Fen won't support anything played from the library, then I'm not understanding what you are asking for.

**Comment by Jm7698** [comment]
If i play a movie or a show in fen, fen doesn't show its watched already or showing progress in fen if watched status provider is set to trakt.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Tikipeter** [comment]
It's supposed to. It does for me.

You've probably got the Kodi Trakt addon installed, and haven't set an exemption for online/addon playback sources.

**Comment by Tikipeter** [comment]
Going right back to your original question, which involved both TMDbHelper and local library playback, you might be better off using a different addon to Fen, instead of going into Kodi's Trakt addon and changing settings.

Umbrella is a Venom fork using basically the same scrapers and is currently being maintained.

**Comment by AltFreq07** [comment]
I too am having an issue with Fen grabbing the progress of Movies and TV Shows
Current in Progress Movies shows all at 0%
![image](https://user-images.githubusercontent.com/36876748/182293226-094e9559-510c-4269-8ce6-ce73235569be.png)
The In Progress Episodes list is empty and watching a new show for more than the Successful playback time does not add it to in progress episodes and starts from the beginning if I play again instead of providing the Resume Playback dialog

**Comment by Jm7698** [comment]
Yes Umbrella works fine, also Ezra. Old fen worked fine with library and tmbhelper.
But, Thank u for developing great addon and i know Ezra is a fork.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by Logs22** [comment]
> I too am having an issue with Fen grabbing the progress of Movies and TV Shows Current in Progress Movies shows all at 0% ![image](https://user-images.githubusercontent.com/36876748/182293226-094e9559-510c-4269-8ce6-ce73235569be.png) The In Progress Episodes list is empty and watching a new show for more than the Successful playback time does not add it to in progress episodes and starts from the beginning if I play again instead of providing the Resume Playback dialog
> 
> I dont have TMDbHelper or the Trakt addon installed

thats a skin issue, known with that skin

**Comment by Jm7698** [comment]
There isn’t any such a options in trakt addon settings. Fen is just not showing progress in kodi library not even in fen addon when in fen options watched status provider selected is trakt. It doesn’t work even if I don’t have a trak adddon installed so it does not anything to do with it.  Its not big deal because other addons shows progress in kodi library.
